src/fping.c

```diff
@@ -1469,6 +1469,15 @@ void print_netdata(void)
     for (i = 0; i < num_hosts; i++) {
         h = table[i];
 
+        /* if we just sent the probe and didn't receive a reply, we shouldn't count it */
+        h->discard_next_recv_i = 0;
+        if (h->waiting && timeval_diff(&current_time, &h->last_send_time) < h->timeout) {
+            if (h->num_sent_i) {
+                h->num_sent_i--;
+                h->discard_next_recv_i = 1;
+            }
+        }
+
         if (!sent_charts) {
             printf("CHART fping.%s_packets '' 'FPing Packets for host %s' packets '%s' fping.packets line 110020 %d\n", h->name, h->host, h->name, report_interval / 100000);
             printf("DIMENSION xmt sent absolute 1 1\n");
```

Comment:
It would be nice you swap `received` and `sent`. It is common pattern in netdata - `sent` goes first and has green color.

Reply:
bad currently: when green > red

after swap: when red > green